aspnetcore/razor-components/index.md

```diff
@@ -48,11 +48,11 @@ For more information, see <xref:razor-components/hosting-models#server-side-host
 
 A *Razor Component* is a piece of UI, such as a page, dialog, or data entry form. Components handle user events and define flexible UI rendering logic. Components can be nested and reused.
 
-Components are .NET classes built into .NET assemblies that can be shared and distributed as NuGet packages. The class can either be written in the form of a Razor markup page (*.cshtml*) or as a C# class (*.cs*).
+Components are .NET classes built into .NET assemblies that can be shared and distributed as NuGet packages. The class is normally written in the form of a Razor markup page with a *.razor* file extension.
```

Comment:
Should we mention that .cshtml is still an option?  Is .cs still an option?

Reply:
Let's pull this here for now given that ...

* The Index is an overview of the suggested approaches, we want devs to use the *.razor* extension for their components.
* The Get Started topic has a note early on this point about authoring with the *.cshtml* extension.

What I need to do (*later*) is flesh this out a bit more in the *Components* topic, including the C# (*.cs*) approach. I've opened https://github.com/aspnet/Docs/issues/11412 to address it when I get back.